package/boot/uboot-rockchip/Makefile

```diff
@@ -118,6 +118,22 @@ define U-Boot/rockpro64-rk3399
 endef
 
 
+# RK3566 boards
+
+define U-Boot/rk3566/Default
+  BUILD_SUBTARGET:=armv8
+  DEPENDS:=+PACKAGE_u-boot-$(1):trusted-firmware-a-rk3568
+  ATF:=rk3568_bl31_ultra_v2.13.elf
+  TPL:=rk3566_ddr_1056MHz_v1.18.bin
+endef
+
+define U-Boot/radxa-cm3-io
```

Comment:
this should be `radxa-cm3-io-rk3566`, which matches the defconfig name.

Reply:
fixed